bindings/wasm/examples-account/src/custom_storage.ts

```diff
@@ -10,17 +10,14 @@ export class MemStore implements Storage {
     // We use strings as keys rather than DIDs or KeyLocations because Maps use
     // referential equality for object keys, and thus a primitive type needs to be used instead.
 
-    // The map from DIDs to chain states.
-    private _chainStates: Map<string, ChainState>;
-    // The map from DIDs to DID documents.
-    private _documents: Map<string, Document>;
+    // The map from DIDs to state.
+    private _state: Map<string, Uint8Array>;
```

Comment:
Nit: maybe this should be `_blobs` to be consistent?

```suggestion
    // Map of DID state blobs.
    private _blobs: Map<string, Uint8Array>;
```